be/src/olap/lru_cache.cpp

```diff
@@ -73,9 +73,6 @@ uint32_t CacheKey::hash(const char* data, size_t n, uint32_t seed) const {
 Cache::~Cache() {}
 
 HandleTable::~HandleTable() {
-    for (uint32_t i = 0; i < _length; i++) {
-        delete _list[i];
```

Comment:
should not delete this code， maybe memory leak here

Reply:
we don't need dummy node for each bucket anymore